src/nIsForName.js

```diff
@@ -22,7 +22,18 @@
  * @return {string[]}
  */
 function nIsForName(names) {
-  // write code here
+  const result = [];
+  for (let i = 0; i < names.length; i++) {
+    const arr = names[i].split(' ');
+    arr.some(function(el, ind) {
```

Comment:
use arrow function on method
```suggestion
    arr.some((el, ind) => {
```